source/databases-collections.txt

```diff
@@ -34,38 +34,35 @@ use the left navigation to:
 Databases and Collections
```

Comment:
[Fix] Maybe, this should be "View Databased and Collections", to make it parallel with the heading below on this same page?
I feel like you left this like that to make it an easy editorial catch :)

Reply:
Agreed. Fixed.